### ISSUE TITLE
could you explain why you are using binary cross-entropy

### ISSUE THREAD

**Issue by xysong1201** [vandit15/Class-balanced-loss-pytorch#8]
Hi, Inside the CBloss you are using binary cross-entropy, so why not using cross-entropy ? Could you explain? Thank you

**Comment by BenDrewry** [comment]
From source: https://arxiv.org/pdf/1901.05555.pdf
"Note that β = 0 corresponds to
no re-weighting and β → 1 corresponds to re-weighing by
inverse class frequency"

It looks like they are using binary CE here to determine which class needs to be scaled appropriately based on the number of effective samples.  So one class will be a 1, while the rest are 0s.
